src/api/api.js

```diff
@@ -22,9 +22,49 @@ module.exports = () => {
         }
     }
 
+    const firstDate = {
```

Comment:
Could these two be in one `array`?
```js
history: [
  {
    date: '12-1-2018', // should serve as a name also
    items: [...] 
  },
  {
    date: '21-2-2018',
    items: [...] 
  }
]
```

Usually BE would provide this as an `array` of `object`s